gitlab/provider.go

```diff
@@ -132,7 +133,7 @@ func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 func validateApiURLVersion(value interface{}, key string) (ws []string, es []error) {
 	v := value.(string)
 	if strings.HasSuffix(v, "/api/v3") || strings.HasSuffix(v, "/api/v3/") {
-		es = append(es, fmt.Errorf("terraform-gitlab-provider does not support v3 api; please upgrade to /api/v4 in %s", v))
+		es = append(es, fmt.Errorf("terraform-provider-gitlab does not support v3 api; please upgrade to /api/v4 in %s", v))
```

Comment:
I can revert this if you want, but I noticed that this line is the only reference of `terraform-gitlab-provider`. All other references use `terraform-provider-gitlab`.